### PR TITLE
Hugh

### PR DESCRIPTION
**Pull request by hnnsgstfssn**
suggested removal etc.

### CODE REVIEW

File: hello [hunk 1]
```diff
@@ -1 +1,3 @@
 world
```
Comment (HughWarrington) [inline hello]: Looks pretty good

File: hello [hunk 2]
```diff
@@ -1 +1,3 @@
...
+
+it's a cruel one.
```
Comment (HughWarrington) [inline hello]: This is now a review

File: hello (hunk 1 above)
Comment (hnnsgstfssn) [reply hello]: i know.

File: hello (hunk 2 above)
Comment (hnnsgstfssn) [reply hello]: ok.

**Pull request merged**